2-transform-data/bq_ingestion/extract_msgs_test.py

```diff
@@ -725,9 +726,51 @@ def test_convert_msg_to_json(self):
             got_json = em.convert_msg_to_json(test["msg"])
             self.assertEqual(want_json[key], got_json, "Convert message to json error")
 
+    
+    @mock.patch("google.cloud.storage.Client")
+    def create_client_mock(self, Client):
+        client = Client()
+
```

Comment:
Simulate any function calls and returns you need from the Client an other objects like `client.get_table(table_id)` would become `client.get_table.return_value = table_id`